mongoimport/mongoimport.go

```diff
@@ -511,6 +513,8 @@ func (imp *MongoImport) importDocument(inserter *db.BufferedBulkInserter, docume
 			result, err = inserter.Insert(document)
```

Comment:
There's a neat little bug where if `--ignoreBlanks` is set, then mongoimport could fallback on insert mode. This causes `--mode delete` to insert documents.

E.g. 

```
./bin/mongoimport -c bar --file ./mongoimport/testdata/test_blanks.csv --type csv --fields a,b,c --ignoreBlanks --mode delete --upsertFields b -v
```

The issue is the `if selector == nil` line. I think the expected behaviour would be to just skip documents where we can't construct the filter.